curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/65688b5a1655a7a6caede847.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 65688b5a1655a7a6caede847
-title: Step 33
+title: Step 31
 challengeType: 20
-dashedName: step-33
+dashedName: step-31
 ---
 
```

Comment:
I would add something to this step:

"When you try to change the individual characters of a string as you did in the previous step, you get a `TypeError`, which occurs when an object of inappropriate type is used in your code.

As you can see from the error message, strings do not support item assignment, because they are immutable. However, a variable can be reassigned another string:"